README.md

```diff
@@ -12,12 +12,12 @@ First you need to clone the repository
 git clone git@github.com:hiero-ledger/hiero-sdk-tck.git
 ```
 
-The TCK provides ready-to-use configurations to run tests against the [Hedera testnet](https://docs.hedera.com/hedera/networks) or [hedera-local-node](https://github.com/hashgraph/hedera-local-node).
+The TCK provides ready-to-use configurations to run tests against [testnet](https://docs.hedera.com/hedera/networks) or [hedera-local-node](https://github.com/hashgraph/hedera-local-node).
```

Comment:
```suggestion
The TCK provides ready-to-use configurations to run tests against the [Hedera testnet](https://docs.hedera.com/hedera/networks) or [hedera-local-node](https://github.com/hashgraph/hedera-local-node).
```
That should stay because in Hiero context "testnet" only makes not much sense